### PR TITLE
Update code to adapt to memory leaks in the editor scene as a prerequisite task

### PR DESCRIPTION
**Pull request by moshuying**
Re: #

### Changelog

* Update code to adapt to memory leaks in the editor scene as a prerequisite task

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Review by moshuying — COMMENTED** [review]
self code review

File: editor/inspector/assets/fbx/preview.js [hunk 1]
```diff
@@ -679,16 +679,16 @@ exports.methods = {
 
     addAssetChangeListener(add = true) {
         if (!add && this.hasListenAssetsChange) {
-            Editor.Message.removeBroadcastListener('asset-db:asset-change', this.onAssetChangeBind);
+            Editor.Message.__protected__.removeBroadcastListener('scene:all-asset-changed', this.onAssetChangeBind);
```
Comment (arsen2010) [inline editor/inspector/assets/fbx/preview.js]: This has a significant impact, which means that the asset change monitoring by the inspector has shifted from the db to the scene.

File: editor/inspector/contributions/asset.js [hunk 2]
```diff
@@ -155,7 +155,7 @@ const Elements = {
                 panel.__assetChangedHandle__ = undefined;
             }
 
-            Editor.Message.removeBroadcastListener('asset-db:asset-change', panel.__assetChanged__);
+            Editor.Message.__protected__.removeBroadcastListener('scene:all-asset-changed', panel.__assetChanged__);
```
Comment (dogeFu) [inline editor/inspector/contributions/asset.js]: 如果说场景之外的逻辑，也需要依赖场景新的assetchange消息，那这个消息应该是放到db那边来实现会好一点。

Comment (arsen2010) [reply editor/inspector/contributions/asset.js]: 这个 PR 对 fbx preview 部分我没意见，但对 asset.js 的调整影响比较大，是不是它可以不用改，很多资源的编辑只是 meta 数据，不需要 scene 的参与

Comment (VisualSJ) [reply editor/inspector/contributions/asset.js]: Here, the inspector dependency on asset-db has been changed to inspector scene.
And the scene manages some of the processes of the resources. So it doesn't feel very logical.

Can you achieve the desired functionality while maintaining the dependency relationship or, in other words, while the resource module still handles resource-related notifications?

Comment (moshuying) [reply editor/inspector/contributions/asset.js]: While it is possible to update data by listening for resource changes, this is costly and expensive to maintain. Therefore, it is now possible to update data more efficiently by reducing the number of non-essential event listeners to scenes and only listening where necessary.

File: editor/inspector/assets/fbx/preview.js (hunk 1 above)
Comment (moshuying) [reply editor/inspector/assets/fbx/preview.js]: its ok, when asset changed,scene update asset,then preview get messge to update preview

**Review by arsen2010 — APPROVED** [review]

**Review by VisualSJ — APPROVED** [review]

**Pull request merged**